src/pybind/mgr/dashboard/services/nvmeof_conf.py

```diff
@@ -82,16 +92,33 @@ def get_service_info(cls, group=None):
             if not gateways:
                 return None
 
-            if group:
-                for service_name, entries in gateways.items():
-                    if group in service_name:
-                        entry = next((entry for entry in entries if entry['group'] == group), None)
-                        if entry['group'] == group:  # type: ignore
-                            return service_name, entry['service_url']  # type: ignore
+            try:
+                if group:
+                    orch = OrchClient.instance()
+
+                    for service_name, entries in gateways.items():
```

Comment:
What's `entries`? We should try to write self-explanatory code.

Reply:
forgot to clean it up. fixed it now